src/tests/test_utils/functional_test_utils/src/ov_plugin_cache.cpp

```diff
@@ -36,66 +91,18 @@ PluginCache& PluginCache::get() {
 std::shared_ptr<ov::Core> PluginCache::core(const std::string& deviceToCheck) {
     std::lock_guard<std::mutex> lock(g_mtx);
```

Comment:
Please move lock_guard below "**if (std::getenv("DISABLE_PLUGIN_CACHE") != nullptr) {...**".
**std::getenv** may take a while. We don't need to wait it under a mutex.

Or you can move **std::getenv** to PluginCache ctr and save result as class field.

Reply:
moved to constructor